### DISCUSSION TITLE
missing autoconf file in installation?

### DISCUSSION THREAD

**Discussion by dkonstrkt** [mikebrady/shairport-sync#1316] [Q&A]
Hello everyone,
I'm a complete n00b with Raspberry Pi and trying to learn. I've followed the directions in _Simple Installation Instructions_ and every time I get stuck at the line where **autoconfi -fi** is used. I'm getting the error "command not found"
Can anyone help me with this? Have I missed something? I'm running a Pi 4 B with IQ AudIO DAC Pro and a newly installed Raspberry Pi OS. I thought that I had followed the directions exactly as they are listed.
Any help would be greatly appreciated!
Thanks,
Marc

**Comment by stephenmetzger** [comment]
Hey Marc. I'm certainly no expert, but have been running Shairport for a while now and would love to try to help you out, just because I think this project is awesome. 

Looks like you may have a typo in the line you mention above. It should be 
```autoreconf -i -f```, 
I think. 
Another thought is that you may have to use 
```sudo autoreconf -i -f```, 
but you'd probably have had to use sudo for the other commands leading up to autoreconf.

**Reply by dkonstrkt** [reply]
Hello Stephen,
Thank you for your reply. I also tried:
autoreconf -i -f, autoreconf -i, autoreconf -f, sudo autoreconf -fi, sudo autoreconf -f, sudo autoreconf -i, sudo autoreconf -i -f

All of them receive _sudo: command not found_ or _bash: command not found_

Just to clarify, I'm following this part of the directions:
          $ git clone https://github.com/mikebrady/shairport-sync.git
          $ cd shairport-sync
          $ **autoreconf -fi**
          $ ./configure --sysconfdir=/etc --with-alsa --with-soxr --with-avahi --with-ssl=openssl --with-systemd
          $ make
          $ sudo make install 

I know I must be missing something simple, but I have no idea what it is.

**Reply by stephenmetzger** [reply]
Have you installed the dependencies? 

Try 

apt install autoconf

**Comment by mikebrady** [comment]
Hi there. Thanks for your interest in Shairport Sync. If you are following the build guide, it's really important to follow the steps _exactly_. That means:

1. You must do the steps in the exact order they are given in. 
2. The commands must be executed exactly as given -- the case of letters is important and you can not substitute I for 1 and O for 0 or anything like that. Spacing is also very important -- don't add or remove spaces from the commands.

So, to answer your question, the command is not `autoconfi -fi`; it is `autoreconf -fi`. The tool should have been installed in one of the earlier steps.

I hope this helps. The Unix command line approach is very powerful but it takes some getting used to.
